vaadin-spring/src/test/java/com/vaadin/flow/spring/service/SpringServletTest.java

```diff
@@ -52,4 +52,13 @@ public void readUniformNameProperty_propertyNameContansDash_propertyNameIsConver
         Assert.assertEquals("someUrl",
                 service.getDeploymentConfiguration().getPushURL());
     }
+
+    @Test
+    public void customPushUrl_rootMapping_isPrefixedWithContextVaadinMapping()
```

Comment:
I know that this didn't even have any test before, but could maybe have another unit test just for the introduced mapping logic with push urls like "/someUrl" and apparently "/vaadinServlet/someUrl".

An IT would be great but it would be rather complex to make happen so I think it can be skipped.

Reply:
Added unit tests for no custom push URL, custom URL with and without preceding slash, as well as custom URL with preceding `/vaadinservlet`.